cmd/notation/list.go

```diff
@@ -55,9 +59,26 @@ func runList(command *cobra.Command, opts *listOpts) error {
 	}
 
 	// write out
-	for _, sigManifest := range sigManifests {
-		fmt.Println(sigManifest.Blob.Digest)
+	return output(manifestDesc.Digest, sigManifests, reference)
+}
+
+func output(digest digest.Digest, sigManifests []notationRegistry.SignatureManifest, reference string) error {
+	ref, err := registry.ParseReference(reference)
+	if err != nil {
+		return err
 	}
 
+	sigCount := len(sigManifests)
+	if sigCount > 0 {
```

Comment:
What's the output if `sigCount == 0`?

/cc @yizha1

Reply:
We don't improve the UX for beta.1. No output means success but no signatures.

Reply:
Consistent with current implementation.